client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js

```diff
@@ -879,7 +886,101 @@ describe( 'getThankYouPageUrl', () => {
 				siteSlug: 'foo.bar',
 			} );
 
-			expect( url ).toBe( '/checkout/offer-professional-email/1234abcd/foo.bar' );
+			expect( url ).toBe(
+				'/checkout/offer-professional-email/domain-eligible-for-free-trial.com/1234abcd/foo.bar'
+			);
+		} );
+
+		it( 'Is displayed if site has eligible domain and Personal plan is in the cart', () => {
+			const cart = {
+				products: [
+					{
+						product_slug: PLAN_PERSONAL,
+					},
+				],
+			};
+
+			const url = getThankYouPageUrl( {
+				...defaultArgs,
+				cart,
+				domains,
+				receiptId: '1234abcd',
+				siteSlug: 'foo.bar',
+			} );
+
+			expect( url ).toBe(
+				'/checkout/offer-professional-email/domain-eligible-for-free-trial.com/1234abcd/foo.bar'
+			);
+		} );
+
+		it( 'Is displayed if site has eligible domain and Business plan is in the cart', () => {
+			const cart = {
+				products: [
+					{
+						product_slug: PLAN_BUSINESS,
+					},
+				],
+			};
+
+			const url = getThankYouPageUrl( {
+				...defaultArgs,
+				cart,
+				domains,
+				receiptId: '1234abcd',
+				siteSlug: 'foo.bar',
+			} );
+
+			expect( url ).toBe(
+				'/checkout/offer-professional-email/domain-eligible-for-free-trial.com/1234abcd/foo.bar'
+			);
+		} );
+
+		it( 'Is displayed if site has eligible domain and eCommerce plan is in the cart', () => {
+			const cart = {
+				products: [
+					{
+						product_slug: PLAN_ECOMMERCE,
+					},
+				],
+			};
+
+			const url = getThankYouPageUrl( {
+				...defaultArgs,
+				cart,
+				domains,
+				receiptId: '1234abcd',
+				siteSlug: 'foo.bar',
+			} );
+
+			expect( url ).toBe(
+				'/checkout/offer-professional-email/domain-eligible-for-free-trial.com/1234abcd/foo.bar'
+			);
+		} );
+
+		it( 'Is displayed if site has domain registration and eligble plan in the cart', () => {
```

Comment:
Typo: eligble => eligible
```suggestion
		it( 'Is displayed if site has domain registration and eligible plan in the cart', () => {
```

Reply:
Good catch!